compiler/src/dotty/tools/dotc/transform/PatternMatcher.scala

```diff
@@ -992,10 +894,7 @@ object PatternMatcher {
     }
 
     val optimizations: List[(String, Plan => Plan)] = List(
-      "hoistLabels" -> hoistLabels,
-      "elimRedundantTests" -> elimRedundantTests,
-      "inlineLabelled" -> inlineLabelled,
-      "mergeVars" -> mergeVars,
+      "mergeTests" -> mergeTests,
```

Comment:
Could the removed optimizations be brought back later? Or are they subsumed by mergeTests ?

Reply:
They are subsumed by `mergeTests`.